dot/state/tries_test.go

```diff
@@ -130,15 +133,20 @@ func Test_Tries_delete(t *testing.T) {
 		t.Run(name, func(t *testing.T) {
 			t.Parallel()
 			ctrl := gomock.NewController(t)
-
-			triesGauge := NewMockGauge(ctrl)
-			triesGauge.EXPECT().Set(testCase.triesGaugeSet)
```

Comment:
please restore the triesGauge expectation here

Reply:
whoops!